mkt/site/tests/__init__.py

```diff
@@ -344,7 +346,87 @@ def login(self, profile):
 JINJA_INSTRUMENTED = False
 
 
-class TestCase(MockEsMixin, RedisTest, MockBrowserIdMixin, test.TestCase):
+class ClassFixtureTestCase(test.TestCase):
+    fixtures = None
+
+    @classmethod
+    def setUpClass(cls):
+        cls.cls_atomics = cls._enter_atomics()
+
+        if cls.fixtures:
+            for db_name in cls._databases_names(include_mirrors=False):
+                    try:
+                        call_command('loaddata', *cls.fixtures, **{
+                            'verbosity': 0,
+                            'commit': False,
+                            'database': db_name,
+                        })
+                    except Exception:
+                        cls._rollback_atomics(cls.cls_atomics)
+                        raise
+        cls.setUpTestData()
+
+    @classmethod
+    def tearDownClass(cls):
+        cls._rollback_atomics(cls.cls_atomics)
+        for conn in connections.all():
+            conn.close()
+
+    @classmethod
+    def _databases_names(cls, include_mirrors=True):
+        # If the test case has a multi_db=True flag, act on all databases,
+        # including mirrors or not. Otherwise, just on the default DB.
+        if getattr(cls, 'multi_db', False):
+            return [alias for alias in connections
+                    if (include_mirrors or
+                        connections[alias].settings_dict['TEST']['MIRROR'])]
+        else:
+            return [DEFAULT_DB_ALIAS]
+
+    @classmethod
+    def _enter_atomics(cls):
```

Comment:
Can you re-arrange the methods order to follow django's ? It's hard to look at the diff between your class and django's right now because the methods aren't in the same order.